selectors/example-selectors.md

```diff
@@ -166,35 +166,49 @@ Selector:
 ```
 
 
-### Getting a full sub-DAG
+### Retrieving data recursively
+
+[UnixFSv1] is a good case of a recursive data structure. Any number of links can be used to create deeply nested structures. The following example is inspired by UnixFSv1, but uses a simplified structure, so that we don't get lost in UnixFSv1 specific implementation details.
```

Comment:
Did you mean to put a link here?

Reply:
!!!  I learned a new feature of markdown on github today...

Reply:
Whaaaat? That's so useful especially if you have so many moving parts as we have on the specs repo. It makes updating links so much easier (and also easier to review).

Reply:
Very -1 on these personally, I prefer not to use these, I hate having them separated. But I'm also fine being consistent when I'm editing a doc that someone else has started with this style.

Reply:
For me it's about being easier to the eye if you read the plain text file. You probably don't care about the exact links.